lib/__tests__/findMatchingFiles-test.js

```diff
@@ -1,12 +1,14 @@
+/* global spyOn */
```

Comment:
Actually I'm a little surprised that the `jest` `env` doesn't give us this. Might be worth looking into that.